src/NZF1.jl

```diff
@@ -0,0 +1,29 @@
+# A. Montoison, Montreal, 04/2018.
```

Comment:
Would you mind adding "Philippe Toint (private communication)" to indicate the provenance?

Reply:
We also need the classification string. See http://www.cuter.rl.ac.uk/Problems/classification.shtml. It would be something like SUR2-AN-V-0.

Reply:
Thanks for the link. I also found SUR2-AN-V-0 for the classification. I added it with the last commit.